js_modules/dagster-ui/packages/ui-core/src/search/BuildAssetSearchResults.tsx

```diff
@@ -0,0 +1,137 @@
+import {COMMON_COLLATOR} from '../app/Util';
```

Comment:
Why is this file here?

Reply:
This is the logic to build the asset filter search results, which are constructed in global search

Reply:
I think we should find a way not to leak this code into OSS since it's for a cloud specific feature but we can look into that later.

Reply:
sounds good